### ISSUE TITLE
amp-lightbox close on ios chrome redirected back to google

### ISSUE THREAD

**Issue by Aivaras100** [ampproject/amphtml#24197]
When close amp-lightbox or amp-sidebar redirected back to Google search results.
Problem detected only on IOS Chrome browser.

When will it be resolved?

**Comment by wassgha** [comment]
Hello, do you have a demo page we can test this on?

**Comment by Aivaras100** [comment]
https://www.google.com/search?q=klipshop+co+uk&rlz=1CDGOYI_enLT743LT743&oq=klipshop+co+uk&aqs=chrome..69i57.7086j0j9&hl=en-GB&sourceid=chrome-mobile&ie=UTF-8

klipshop co uk 
open page then open menu and close.
Width OS Chrome

**Comment by Aivaras100** [comment]
1. Open this url width Iphone Chrome browser: https://www.google.com/search?q=klipshop+co+uk&rlz=1CDGOYI_enLT743LT743&oq=klipshop+co+uk&aqs=chrome..69i57.7086j0j9&hl=en-GB&sourceid=chrome-mobile&ie=UTF-8
and click on klipshop.co.uk page  in search results (e.g.: https://prnt.sc/oysdlq )
2. In the opening klipshop.co.uk amp page open menu (e.g.: https://prnt.sc/oyseyl)
3. Then click on close menu button (e.g.: https://prnt.sc/oysfya) 

Then you redirected back to google search result :(

**Comment by wassgha** [comment]
Chrome on iOS shows
```
ERRORfailed to execute a task:: Request popHistory failed: Error: ga
ERRORhistory trace for pop: History: failed to execute a task:: Request popHistory failed: Error: ga _reported_
```

**Comment by dvoytenko** [comment]
@wassgha Let's forward this to Google Search team.

**Comment by Aivaras100** [comment]
Analogical problem width amp-lightbox component :(

**Comment by Aivaras100** [comment]
How are the problems solved?

**Comment by Aivaras100** [comment]
Why is the task closed ???  Problem not solved!

**Comment by wassgha** [comment]
Hello @Aivaras100 , following the steps you have mentioned above I was not able to reproduce this on Chrome iOS anymore, I've also tried multiple other websites that had the same issue before and it seems to have been fixed. Could you kindly try to reproduce on your side and let us know if this is still an issue?

**Comment by Aivaras100** [comment]
Updating the browser fixes the issue.  How long before the problem was resolved? 
what browser version is not a problem?

**Comment by wassgha** [comment]
This seems to have been fixed in Chrome 78 according to the changelog. Not an AMP issue anyway.
